web/src/layout/common/FilesModal.tsx

```diff
@@ -225,7 +225,7 @@ const FilesModal = (props: Props) => {
           breakPoint="md"
         >
           <div className="h-100 mw-100">
-            <div className="d-flex flex-row align-items-strecht g-0 h-100 mh-100">
+            <div className="d-flex flex-row align-items-stretched g-0 h-100 mh-100">
```

Comment:
I'm not at all certain what `strecht` was. It might be `stretch`.

Reply:
My fault, we can use `align-items-stretch` in all cases, thanks

https://getbootstrap.com/docs/5.1/utilities/flex/#align-items